package.json

```diff
@@ -2,11 +2,11 @@
   "name": "PXBlueColors",
   "version": "2.0.0",
   "description": "PX Blue colors for Eaton applications",
-  "main": "index.js",
+  "main": "index.ts",
   "scripts": {
     "test": "./buildTest.sh",
-    "build:colors": "cd ui && yarn build && cd ..",
-    "build:colors-branding": "cd branding && yarn build && cd ..",
+    "build:colors": "cd ui && yarn && tsc && cd ..",
+    "build:colors-branding": "cd branding && yarn && tsc && cd ..",
```

Comment:
```suggestion
    "build:colors-branding": "cd branding && yarn build && cd ..",
```

Reply:
the build command (updated) will do yarn install and tsc, so we can leave it as is